Bugzilla/Config.pm

```diff
@@ -249,72 +276,72 @@ sub update_params {
 }
 
 sub write_params {
-  my ($param_data) = @_;
-  $param_data ||= Bugzilla->params;
-
-  local $Data::Dumper::Sortkeys = 1;
-
-  my %params = %$param_data;
-  $params{urlbase} = Bugzilla->localconfig->urlbase;
-  __PACKAGE__->_write_file(Data::Dumper->Dump([\%params], ['*param']));
+  my ($params) = @_;
+  $params ||= Bugzilla->params;
+
+  unless (Bugzilla->request_cache->{no_database}) {
+    try {
+      foreach my $key (keys %{$params}) {
+        if (my $param = Bugzilla::Config::Param->new({name => $key})) {
+          my $value = $params->{$key} || ($param->is_numeric ? 0 : '');
```

Comment:
Oh, I thought that after [this commit](https://github.com/mozilla-bteam/bmo/pull/1765/commits/57bd5ff7d5f35b20e2ebeb4b8020d902cbf56d63), we were dropping the `is_numeric` part of parameters. Can you clarify why it's been added back?

Reply:
New changes do drop is_numeric and just overwrite current value always without needed to do the comparison.